tests/pytorch/testproblems/test_cifar10_resnet32nobn.py

```diff
@@ -0,0 +1,128 @@
+"""Test the CIFAR-10 problem with the ResNet32 (BN replaced by Identity).
+
+Check that BackPACK's extensions and ViViT works with this architecture. 
+Also check compatibility with DeepOBS by running the StandardRunner.
+"""
+
+from copy import deepcopy
+
+import torch
+from backpack import backpack, extend
+from backpack.core.derivatives.convnd import weight_jac_t_save_memory
+from backpack.extensions import BatchGrad, DiagGGNExact
```

Comment:
Is this test executed by GitHub actions? If so, BackPACK should be added as dependency

Reply:
I think we didn't set up GitHub actions for DeepOBS. Or will the ViViT GitHub actions also execute tests that refer to DeepOBS?